### PR TITLE
[PFSP2-0006] Privacy Policy

### LINKED ISSUE #63: [PFSP2-0006] Privacy Policy
### Priority
- [x] Low
- [ ] Mid
- [ ] High

### Links
Name | Link
-|-
User Story | [link]()
Figma | [link](https://www.figma.com/file/YWJimHMl9fq3v4dQBZSuQo/Hi-Fi-User?node-id=393%3A5454)

### Completion
Expected | Time
-|-
Expected Start | 01-09-2020
Expected End | 29-09-2020

image

### PR DESCRIPTION
**Pull request by SetyaG**
Closes #63 
***
## Description
Create the Privacy Policy page

## Related Links
List related PRs against other branches:

List | link
------ | ------
User Story | [link]()
Figma | [link]()
Issue | [link](#63 )

## Todos
- [X] UI
- [ ] Coordinator
- [ ] Network
- [X] Tests
- [ ] Documentation

## Screenshot / Video
Screenshot consisting of changes made on this branch.

### UI Changes
UI related changes will be shown here (can be either screenshot or video format)

### Network Status
Network related changes will be shown here (screenshot of request status)

### Test Coverage
Test coverage result for designated changes will be shown here (screenshot of coverage)
Screen Shot 2020-09-21 at 11 05 14

## Notes
Additional notes that you have either technical debt, blocker, etc.

### CODE REVIEW

**Review by inxudianz — CHANGES_REQUESTED** [review]
Please provide Gif of the finished product

File: MainApp/MainApp/Feature/Profile/PrivacyPolicy/PrivacyPolicyViewController.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+//
+
+import UIKit
+import Component
+
+class PrivacyPolicyViewController: MasterViewController, PrivacyPolicyViewProtocol {
+    
+    // MARK: - Outlet
+    @IBOutlet weak var PrivacyPolicyScrollView: UIScrollView!
+    @IBOutlet weak var PrivacyPolicyLabel: UILabel!
```
Comment (inxudianz) [inline MainApp/MainApp/Feature/Profile/PrivacyPolicy/PrivacyPolicyViewController.swift]: Please use the designated font

**Review by inxudianz — CHANGES_REQUESTED** [review]
Please provide Gif of the finished product

File: MainApp/MainApp/Feature/Profile/PrivacyPolicy/PrivacyPolicyViewController.swift (hunk 1 above)
Comment (SetyaG) [reply MainApp/MainApp/Feature/Profile/PrivacyPolicy/PrivacyPolicyViewController.swift]: done

**Comment by SetyaG** [comment]
privacy policy

**Pull request merged**